### PR TITLE
agent-js: use Node 8 in Docker images

### PR DESCRIPTION
**Pull request by conord33**
This change is [Reviewable](https://reviewable.io/reviews/stratumn/indigo-js/139)

### CODE REVIEW

File: packages/agent-js/Dockerfile.development.alpine [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM mhart/alpine-node:6.7.0
```
Comment (conord33) [inline packages/agent-js/Dockerfile.development.alpine]: not sure if there was a reason to be using this alpine node instead of the official version

**Comment by t-bast** [comment]
Interesting, I don't know indeed if there's a reason for that...you'll probably want to chat with Adrien or Stephan to get a sign-off on that to see if they had a reason to use that alpine image, otherwise :lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/stratumn/indigo-js/139#-:-KzTwVrEkKaPU0a3Ik34:b38e11a)*

**Comment by stephan83** [comment]
It's because the official image was several hundred megabytes.

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/stratumn/indigo-js/139#-:-KzU-bXIpmL2oTUdvLZN:bi5klxr)*

**Comment by conord33** [comment]
Looks like the alpine differences are a lot smaller now so I think we should stick with the official release if that was the only issue.
```
node 8.9.1-alpine  67.7MB
mhart/alpine-node 8.9.1  66.3MB
```

**Pull request merged**